interp/dumpglob.mli

```diff
@@ -8,6 +8,95 @@
 (*         *     (see LICENSE file for the text of the license)         *)
 (************************************************************************)
 
+(** This file implements the Coq's [.glob] file format, which
+   provides information about the objects that are defined and referenced
+   from a Coq file.
+
+   The [.glob] file format is notably used by [coqdoc] and [coq2html]
+   to generate links and other documentation meta-data.
+
+   Note that we consider this format a legacy one, and no stability
+   guarantees are provided as of today, as we search to replace this
+   format with a more structured and strongly-typed API.
+
+   However, we do provide up to date documentation about the format of
+   [.glob] files below.
+
+*)
+
+(** {2 The [.glob] file format}
+
+   [.glob] files contain a header, and then a list of entries, with
+  one line per entry.
+
+  {3 [.glob] header }
+
+  The header consists of two lines:
+
+[DIGEST: %md5sum_of_file]
+[%modpath]
+
+  where %modpath is the fully-qualified module name of the library that the
+  [.glob] file refers to. [%md5sum_of_file] may be NO if [-dump-glob file] was used.
+
+ {3 [.glob] entries }
+
+  There are 2 kinds of [.glob] entries:
+
+  - *definitions*: these entries correspond to definitions of inductives,
+    functions, binders, notations. They are written as:
+
+    [%kind %bc:%ec %secpath %name]
+
+    where [%kind] is one of
+    [{def,coe,subclass,canonstruc,ex,scheme,proj,inst,meth,defax,prfax,thm,prim,class,var,indrec,rec,corec,ind,variant,coind,constr,not,binder}],
```

Comment:
What about lib and mod (is there also modtype?)

Reply:
I guess
https://github.com/coq/coq/blob/d9d51353214eb04dea89285b1a8713a92b9d82b2/vernac/vernacentries.ml#L1461
several "mod" and "modtype" in the same file

Reply:
Thanks, added as best as I could.